### ISSUE TITLE
Older issues that reference newer issues don't automatically link

### ISSUE THREAD

**Issue by magreenblatt** [jeffwidman/bitbucket-issue-migration#143]
When importing an issue, if it references a newer (not yet imported) issue, then comments such as "Duplicate of #XXX" don't become clickable links to issue XXX. Editing and re-saving the comment later, after issue XXX is imported, does make it clickable. (Note: I'm not sure we can/should do anything about this behavior, just pointing it).

**Comment by jeffwidman** [comment]
Yeah, not really sure there's much we can about this without adding a post-processing loop...
